grpc.echo/zilla.yaml

```diff
@@ -5,7 +5,7 @@ catalogs:
     options:
       subjects:
         echo:
-          path: proto/echo.proto
+          path: /proto/echo.proto
```

Comment:
Is it possible to keep this as a relative path to `zilla.yaml`?
We have been attempting to keep the configuration relocatable, anchored by `zilla.yaml` location as a best practice, then switching to remote configuration does not require updating the contents of `zilla.yaml`.

Same feedback applies to other modified `zilla.yaml` configs.

Reply:
Static root paths are more common in containerized runtimes and usually the closer to the root dir the better since these aren't normal linux environments and it makes people lives easier when all of the relevant files aren't in a lot of different places. 

This example is what would happen in production systems that don't use remote schema registries where they may have multiple microservices all using to the same file mounted resource. From the DevOps perspective it is more immutable to maintain a file in the same file mount location and configure the different services to point to the common location. 

The solution put in place previous would require either looking up where the root directory for zilla is in the container root, which could change. The manual configuring of each file in the specific value.yaml scheme in place is also not very portable to other deployable services.

One alternative is to put them in a designated system folder like `/var` or `/usr` if that makes a difference for security. And if it is simply better to have the `zilla.yaml` in the same location then we can configure that as well.

Reply:
Yep, agreed regarding `/etc` per standard Linux filesystem hierarchy.

In our docker container `zilla.yaml` is expected in `/etc/zilla/zilla.yaml` and other resources are relative to that in `/etc/zilla` by default.

In this specific case, `proto` files used by `zilla` would be in `/etc/zilla/proto/` then.

Prefer we keep all the zilla config in `/etc/zilla/` or subdirectories if feasible.

Reply:
`/etc/zilla/` does make sense and is actually required for the `zilla.yaml` can be executed by the `zilla` cli. I have updated the PR to mount there specifically. 

I tested these with the `-c config` from outside of the project folder, and it works, so that is a win!